js/index.js

```diff
@@ -1 +1,31 @@
-//Implement the red light using jQuery. Don't forget to add the script tags.
\ No newline at end of file
+//Implement the red light using jQuery. Don't forget to add the script tags.
+//test
+$(document).ready(function(){
+
+
+  $('#stopButton').click(turnRed);
+  $('#slowButton').click(turnYellow);
+  $('#goButton').click(turnGreen);
+
+  function turnRed() {
+
+    $("#stopLight").css('backgroundColor', 'red');
+    $("#slowLight").css('backgroundColor', 'black');
```

Comment:
In each of these functions you manually change the background color to black. It is general better to have that functionality in a function and call it when needed.